server/services/message-service.js

```diff
@@ -1,5 +1,6 @@
 const auth = require('@azure/ms-rest-nodeauth')
 const config = require('../config')
+const mqConfig = config.messageQueues
```

Comment:
I went for this as it meant lines 12, 13, 15 could use `mqConfig` rather than `config.messageQueues`.
I didn't require `mq-config` as `config` is still needed for `isProd` and thought it was better to use the already required object than require another.